datanode/docker/Dockerfile_storageapi

```diff
@@ -16,7 +16,7 @@
 
 # To build an image from this Dockerfile:
 # docker image build -f Dockerfile_storageapi -t interussplatform/storage_api:tcl4 ../src
-# docker tag interussplatform/storage_api:tcl4 interussplatform/storage_api:tcl4.1.0.011
+# docker tag interussplatform/storage_api:tcl4 interussplatform/storage_api:tcl4.1.1.012
```

Comment:
I really don't like using the tags and updating every time (including updating the startup script on the nodes). Can we do a latest tag and a version tag? That way everything points to latest and you can manually go to a specific version if needed.

Reply:
Why not?  A tag is just tiny metadata that names a particular docker layer stack, and it seems like it should be very useful to be able to go back to any previously-used docker image.  Keeping every version fixes the "well it worked fine before; put it back like it was" problem.  The image is still built and published with the tag "interussplatform/storage_api:tcl4"; this line merely also publishes it as "interussplatform/storage_api:tcl4.1.1.012" (and actually, it doesn't publish; that requires an undocumented docker push since only the interussplatform docker hub admins can do that).  This is how most docker repositories are done; for instance at https://hub.docker.com/_/zookeeper/, the tags "latest", "3.4", and "3.4.13" all point to the same image.  Later, "3.4.13" will still point to the same image, but "3.4" will point to, perhaps, "3.4.14".